buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -153,7 +153,21 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
          's390x_linux_aot'    : CURRENT_RELEASES,
          'x86-64_linux_aot'   : CURRENT_RELEASES,
          'x86-64_mac_aot'     : CURRENT_RELEASES,
-         'x86-64_windows_aot' : CURRENT_RELEASES]
+         'x86-64_windows_aot' : CURRENT_RELEASES
+         'ppc64le_linux_valhalla'   : ['next'],
+         'ppc64le_linux_vt_standard' : ['next'],
+         'aarch64_linux_valhalla'   : ['next'],
+         'aarch64_linux_vt_standard' : ['next'],
+         'x86-64_mac_valhalla'   : ['next'],
+         'x86-64_mac_vt_standard' : ['next'],
+         's390x_zos_valhalla'   : ['next'],
+         's390x_zos_vt_standard' : ['next'],
```

Comment:
I think we only have 11 for z/os at the moment so I'm not sure if these 2 lines are req'd. Won't hurt anything but may not work if you try it.